docs/books/user-guide/theory_of_operation.adoc

```diff
@@ -260,26 +260,25 @@ Anycast distribution delivers each message to one consumer whereas
 multicast distribution delivers each message to all consumers.
 
 Anycast delivery is reliable when the message deliveries are
-unsettled.  There is a reliability contract that the router network
+unsettled. There is a reliability contract that the router network
 abides by when delivering unsettled messages to anycast addresses.
 For every such delivery sent by a producer, the router network
 guarantees that one of the following outcomes will occur:
 
-* The delivery shall be settled with ACCEPTED or REJECTED disposition
-  where the disposition is supplied by the consumer.
-* The delivery shall be settled with RELEASED disposition, meaning
-  that the message was not delivered to any consumer.
-* The delivery shall be settled with MODIFIED disposition, meaning
-  that the message may have been delivered to a consumer but should be
-  considered in-doubt and re-sent.
-* The connection to the producer shall be dropped, signifying that all
-  unsettled deliveries should now be considered in-doubt by the
-  producer and later re-sent.
-
-Multicast delivery is not reliable.  If a producer sends an unsettled
-delivery, the ingress router shall settle the delivery with ACCEPTED
-disposition regardless of whether the message was delivered to any
-consumers.
+* The delivery shall be settled with ACCEPTED or REJECTED disposition where the disposition is supplied by the consumer.
+
+* The delivery shall be settled with RELEASED disposition, meaning that the message was not delivered to any consumer.
+
+* The delivery shall be settled with MODIFIED disposition, meaning that the message may have been delivered to a consumer but should be considered in-doubt and re-sent.
+
+* The connection to the producer shall be dropped, signifying that all unsettled deliveries should now be considered in-doubt by the producer and later re-sent.
+
+Multicast delivery is not reliable. If a producer sends an unsettled
+delivery, one of the following outcomes will occur:
+
+* If the address has no consumers, the router sends back a disposition of RELEASED.
+
+* If the address has one or more consumers, the router sends back a disposition of SETTLED.
```

Comment:
SETTLED is not a disposition.  A delivery may be settled or unsettled, regardless of disposition.  The possible dispositions for a delivery are ACCEPTED, REJECTED, RELEASED, and MODIFIED.